core/src/components/searchbar/usage/angular.md

```diff
@@ -43,3 +43,13 @@
   <ion-searchbar></ion-searchbar>
 </ion-toolbar>
 ```
+
+### Showing the proper keyboard submit button
```

Comment:
```suggestion
### Keyboard Display
```

Something shorter like this

Reply:
I don't think we need this in every usage section since it doesn't really change per framework, probably just do one paragraph on it in the main README